src/dotnet-scaffolding/dotnet-scaffold/Helpers/ToolHelper.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Reflection;
+
+namespace Microsoft.DotNet.Tools.Scaffold.Helpers;
+
+internal static class ToolHelper
+{
+    // Define keywords for prerelease versions
+    internal static string[] PrereleaseKeywords = { "preview", "rc", "rtm", "alpha", "beta", "dev" };
```

Comment:
_Probably_ not worth bringing in NuGet.Versioning package just for this, but if we did (and I don't think it has any transitive dependencies, so not that big of a deal), SemanticVersion class has a [IsPrerelease](https://github.com/NuGet/NuGet.Client/blob/dddfdd8617e80ec5a2ea0dd94a094b243f355d09/src/NuGet.Core/NuGet.Versioning/SemanticVersion.cs#L213) property.

Reply:
not worth it IMO, have used it in the past and it can bug out.